src/Ilios/CoreBundle/Controller/OfferingController.php

```diff
@@ -251,6 +282,22 @@ public function putAction(Request $request, $id)
 
             $this->getOfferingHandler()->updateOffering($offering, true, true);
 
+            if (Codes::HTTP_CREATED === $code) {
```

Comment:
It looks to me like I2 only created a new offering alert for cases where the session is published, but this adds an alert for every new offering.
https://github.com/ilios/ilios/blob/2.4.x/web/application/controllers/offering_management.php#L333